task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskAssignment.vue

```diff
@@ -120,7 +122,12 @@
           placeholder: this.$t('label.assignee'),
           noDataLabel: this.$t('label.noDataLabel'),
         };
-      }
+      },
+      searchOptions() {
+        return {
+          projectId: this.task.status.project.id
```

Comment:
Put the full URL including the project ID, the suggester should not know what is the application calling, it should just know the URL to use for searching users.